src/Workflow.php

```diff
@@ -387,6 +387,46 @@ public static function registerSignal(string $queryType, callable $handler): Sco
         return self::getCurrentContext()->registerSignal($queryType, $handler);
     }
 
+    /**
+     * Registers an Update method in the Workflow.
+     *
+     * ```php
+     *  #[WorkflowMethod]
+     *  public function handler()
+     *  {
+     *      Workflow::registerUpdate(
```

Comment:
I think it's a little strange for the example to be _registering_ the handler in a function called `handler` which is also the workflow method. Maybe just call the function `exampleWorkflow` or something.

Reply:
Removed the function wrapper for all Workflow::register* methods.